frontend/app/(dashboard)/updates/company/Edit.tsx

```
@@ -0,0 +1,201 @@
+"use client";
+
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/16/solid";
+import { EnvelopeIcon, UsersIcon } from "@heroicons/react/24/outline";
+import { useMutation } from "@tanstack/react-query";
```

ESLint

[error] There should be no empty line between import groups

(import/order)